docs/cohorts.md

```diff
@@ -8,23 +8,94 @@ hide_title: true
+
+OR
+
+* Procedure code in [Intravesical Administration of BCG Procedure VS](/docs/vocabulary#intravesical-administration-of-bcg-procedure-vs)
+
+OR
+
+* Procedure code in [Intravesical Administration of Therapeutic Agent Procedure VS](/docs/vocabulary#intravesical-administration-of-therapeutic-agent-procedure-vs) AND
+* Procedure itemUsed in [BCG Vaccine Products for Intravesical Use VS](/docs/vocabulary#bcg-vaccine-products-for-intravesical-use-vs)
+
+### Inpatient Convalescent Serum of COVID-19 Use
+
+Use of convalescent serum of COVID-19 (CS) during the course of hospitalization. Includes any dose or duration of CS, any level of disease severity while on CS, and patients on other medications.
+
+* Medication administration code in [Convalescent Serum Products VS](/docs/vocabulary#convalescent-serum-of-covid-19-products-vs) during [COVID-19 Related Hospitalization](#covid-19-related-hospitalization)
+
+### Inpatient Death
+
+Patient who dies during hospitalization:
+
+* Discharge disposition like `expired` OR date of death during hospitalization
+
+### Inpatient HCQ Use
+
+Use of hydroxychloroquine during a hospitalization related to COVID-19. Includes any dose, form, duration of HCQ, any level of disease severity while on HCQ, and patients on other medications.
+
+* Medication administration code from [Hydroxychloroquine VS](/docs/vocabulary#hydroxychloroquine-vs) during [COVID-19 Related Hospitalization](#covid-19-related-hospitalization)
+
+### Inpatient Remdesivir Use
+
+Use of Remdesivir during a hospitalization related to COVID-19. Includes any dose, form, duration of Remdesivir, any level of disease severity while on Remdesivir, and patients on other medications.
+
+* Medication administration code from [Remdesivir VS](/docs/vocabulary#remdesivir-vs) during [COVID-19 Related Hospitalization](#covid-19-related-hospitalization)
+
+### Invasive Mechanical Ventilation
+
+Any intubation procedure or mechanical ventilation-associated procedure, excluding supplemental oxygen (high or low flow), CPAP, and BiPAP, ECMO.
+
+* Procedure code in [Invasive Mechanical Ventilation VS](/docs/vocabulary#invasive-mechanical-ventilation-vs) OR
+* Evidence of mechanical ventilation in other intensive care unit (ICU) documentation (e.g. ventilator mode change in ICU flowsheet)
+
+### Prior HCQ Use
+
+Use of hydroxychloroquine prior to hospital admission related to COVID-19:
+
+* Medication code from [Hydroxychloroquine VS](/docs/vocabulary#hydroxychloroquine-vs) on the admission medication list or outpatient medication list associated with [COVID-19 Related Hospitalization](#covid-19-related-hospitalization)
+
+### Respiratory Condition
+
+Patient exhibiting signs and symptoms of respiratory distress:
+
+* Condition code, encounter admitting diagnosis code, or encounter chief complaint code in [Respiratory Condition VS](/docs/vocabulary#respiratory-condition-vs) value set.
```

Comment:
Why are you singling out admitting Dx code and chief complaint from Condition code? They're also condition codes. If anything we want to differentiate billing and clinical condition codes, of of the latter, encounter codes from PL items.

Reply:
fixed.